feature/bgp/tests/local_bgp_test/local_bgp_test.go

```diff
@@ -236,7 +236,7 @@ func TestParameters(t *testing.T) {
 				PeerAs:          ygot.Uint32(ateAS),
 				NeighborAddress: ygot.String(ateIP),
 				Timers: &telemetry.NetworkInstance_Protocol_Bgp_Neighbor_Timers{
-					HoldTime:          ygot.Uint16(100),
+					HoldTime:          ygot.Uint16(150),
```

Comment:
If there is a relationship, could we make them a constant like this?

```
const (
  keepAlive = 50
  holdTime = keepAlive * 3 // Must be at least 3 times, see (reference).
)
```

If there is a source that explains why this has to be case, let's add it?

Reply:
I added a const for the values.